graphql_api/types/test_results_headers/__init__.py

```diff
@@ -0,0 +1,11 @@
+from shared.license import get_current_license
```

Comment:
Do we need this license stuff? think I wrote this on your other PR but lets clean it up here